meta-lmp-base/recipes-bsp/u-boot/u-boot-fio-mfgtool/lmp-common.cfg

```diff
@@ -0,0 +1,103 @@
+CONFIG_BOOTDELAY=-2
+# CONFIG_CMD_BOOTZ is not set
+CONFIG_FIT_SIGNATURE_STRICT=y
+CONFIG_FIT_SIGNATURE=y
+CONFIG_FIT_VERBOSE=y
+CONFIG_FIT=y
+# CONFIG_LEGACY_IMAGE_FORMAT is not set
+# CONFIG_NET is not set
+CONFIG_RSA=y
+CONFIG_USE_BOOTCOMMAND=y
+CONFIG_BOOTCOMMAND="reset"
+# CONFIG_BOOTM_NETBSD is not set
+# CONFIG_BOOTM_PLAN9 is not set
+# CONFIG_BOOTM_RTEMS is not set
+# CONFIG_BOOTM_VXWORKS is not set
+# CONFIG_CMD_BDI is not set
+# CONFIG_CMD_BMODE is not set
+# CONFIG_CMD_BOOTD is not set
+# CONFIG_CMD_DEKBLOB is not set
+# CONFIG_CMD_EDITENV is not set
+# CONFIG_CMD_EXPORTENV is not set
+CONFIG_CMD_FIOVB=y
+# CONFIG_CMD_IMPORTENV is not set
+# CONFIG_CMD_SATA is not set
+# CONFIG_CMD_SYSBOOT is not set
+# CONFIG_CMD_USB_MASS_STORAGE is not set
+CONFIG_CMD_XIMG=y
+# CONFIG_DISTRO_DEFAULTS is not set
+# CONFIG_DM_SCSI is not set
+# CONFIG_DWC_AHSATA is not set
+# CONFIG_EFI_LOADER is not set
+CONFIG_ENV_IS_NOWHERE=y
+CONFIG_FASTBOOT_FLASH=y
+CONFIG_FIOVB=y
+CONFIG_FSL_FASTBOOT_BOOTLOADER_SECONDARY=y
+CONFIG_OF_LIBFDT_OVERLAY=y
+CONFIG_OPTEE_LIB=y
+CONFIG_OPTEE=y
+CONFIG_PARTITION_UUIDS=y
+# CONFIG_SATA is not set
+CONFIG_SPL_CRYPTO=y
+# CONFIG_SPL_EFI_PARTITION is not set
+CONFIG_SPL_FIT_SIGNATURE=y
+CONFIG_SPL_HASH=y
+# CONFIG_SPL_LEGACY_IMAGE_SUPPORT is not set
+CONFIG_SPL_LOAD_FIT=y
+# CONFIG_SPL_PARTITION_UUIDS is not set
+CONFIG_TEE=y
+CONFIG_USB_FUNCTION_FASTBOOT=y
+# CONFIG_USB_HOST_ETHER is not set
+# CONFIG_USB_KEYBOARD is not set
+# CONFIG_USB_STORAGE is not set
+CONFIG_DM_USB=y
+# CONFIG_ENV_IS_IN_MMC is not set
+CONFIG_FASTBOOT_UUU_SUPPORT=y
+CONFIG_FSL_FASTBOOT_BOOTLOADER2=y
+CONFIG_SPL_DM=y
+# CONFIG_SPL_DOS_PARTITION is not set
+CONFIG_SPL_OF_CONTROL=y
+CONFIG_SPL_USB_GADGET=y
+CONFIG_SPL_USB_SDP_SUPPORT=y
+CONFIG_USB_GADGET_MANUFACTURER="FSL"
+CONFIG_USB_GADGET_PRODUCT_NUM=0x0152
+CONFIG_USB_GADGET_VENDOR_NUM=0x1fc9
+CONFIG_USB=y
+# CONFIG_ANDROID_BOOT_IMAGE is not set
+# CONFIG_CMD_BOOTP is not set
+# CONFIG_CMD_DHCP is not set
+# CONFIG_CMD_NET is not set
+# CONFIG_CMD_NFS is not set
+# CONFIG_CMD_PXE is not set
+# CONFIG_CMD_SAVEENV is not set
+# CONFIG_CMD_TFTPBOOT is not set
+# CONFIG_DM_VIDEO is not set
+# CONFIG_SHA_HW_ACCEL is not set
+# CONFIG_SPL_DM_USB is not set
+# CONFIG_CMD_ELF is not set
+# CONFIG_CMD_FLASH is not set
+# CONFIG_CMD_IMI is not set
+# CONFIG_CMD_LOADB is not set
+# CONFIG_CMD_LOADS is not set
+# CONFIG_CMD_GETTIME is not set
+# CONFIG_CMD_POWEROFF is not set
+# CONFIG_CMD_RTC is not set
+# CONFIG_CMD_SNTP is not set
+# CONFIG_CMD_TIME is not set
+# CONFIG_CMD_TIMER is not set
+# CONFIG_DFU is not set
+# CONFIG_ENV_IS_IN_SPI_FLASH is not set
+CONFIG_SPL_FIT_SIGNATURE_STRICT=y
+# CONFIG_SPL_MMC is not set
```

Comment:
Nit, can we sort the options to gather the SPL related ones together? Easier to read and potentially debug later (which are usually the options we want to carefully review due size restrictions).